scripts/release.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# Copyright (c) The Diem Core Contributors
```

Comment:
idk what the consensus was on the copyright headers but i don't think you need the Diem ones anymore?

Reply:
The CI check prompt requires a copyright notice, I copied a copy from git-checks.sh

https://github.com/move-language/move/blob/548f632ca3ace2707c59c96dda198a9a7bead592/scripts/git-checks.sh#L2